spec/models/profile_presenter_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'spec_helper'
+
+describe ProfilePresenter do
+  let(:user) { create(:user, email: "example@example.com") }
+  let(:profile) { user.create_profile }
+  let(:preference) { user.create_preference }
```

Comment:
Why `create_profile`/`create_preference` here, instead of `profile` and `preference`? They should have been created by the callbacks in the user class:
https://github.com/otwcode/otwarchive/blob/86d9cfcb409ff316ef2ffa9ad2188241e151abf9/app/models/user.rb#L284-L288